contracts/modules/capital/Pool.sol

```diff
@@ -34,6 +37,21 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
   IPriceFeedOracle public override priceFeedOracle;
   address public swapOperator;
 
+  // Binary map where each on bit, starting from the LSB, represents whether the payout asset found
+  // at the same index as the bit's position should be ignored when calulating the value of the pool
+  // in ETH.
+  //
+  // Examples:
+  // 1 (10) = 00000000000000000000000000000001 (2)
+  //                                         ^
+  //                                         payoutAssets[0] is ignored
+  //
+  // 9 (10) = 00000000000000000000000000001001 (2)
+  //                                      ^  ^
+  //                                      payoutAssets[0] and payoutAssets[3] are both ignored
+  //
+  uint32 public ignoredPayoutAssetsBitmap;
```

Comment:
It seems to me that this flag is only used to mark the "payout" assets as being deprecated and got me initially confused to think that it means that the asset is present in both arrays and should be skipped when iterating over one of them. If this flag's only purpose is deprecation I'd suggest calling it `deprecatedPayoutAssetsBitmap` or `deprecatedCoverAssetsBitmap` if we proceed with the payout -> cover rename.